README.md

```diff
@@ -1,5 +1,10 @@
 <p align="center"><img width="40%" src="docs/ONNXMLTools_logo_main.png" /></p>
 
+[![TravisCI](https://travis-ci.org/xadupre/onnxmltools.svg)](ttps://travis-ci.org/xadupre/onnxmltools)
+[![CircleCI](https://circleci.com/gh/xadupre/onnxmltools.svg?style=svg)](https://circleci.com/gh/xadupre/onnxmltools)
```

Comment:
May be remove CircleCI and just keep TravisCI since both do the same thing but ONNX repo also uses Travis that way we will be consistent.

Reply:
Ok, I'll remove.

Reply:
Thanks, Xavier!